package-lock.json

```diff
@@ -8515,11 +8515,6 @@
       "integrity": "sha512-YJhbp0sz3egFFKl3BcCNPQKzuGFOP4PACcsifhK6ROGnJUW9ViYLuLybQ9GQZm7Zejy3tkGuiXYMq3GiyGkU4g==",
       "dev": true
     },
-    "@types/mime-types": {
```

Comment:
@kostyalmm seems like `package-lock.json` has been updated without dependencies change in the `package.json` file. Could you please revert changes in the `package-lock.json`?